app/models/host.rb

```diff
@@ -59,6 +59,17 @@ def readonly?
     true
   end
 
+  def self.os_version_query(path, values, include = true)
+    query = include ? :in : :not_in
+    path = %i[major minor].include?(path) && path
```

Comment:
What would happen with `false` when `path` does not match allowed values? Can it raise exception early in the process to signal a code mistake?